README.md

```diff
@@ -1,5 +1,14 @@
 # Ngenic Sensor Platform for Home Assistant
-This platform adds sensors for Ngenic Tune nodes that report temperature and humidity.
+This platform adds sensors for Ngenic Tune nodes that report temperature and humidity if a Ngenic Track device is connected the current power and energy consumption for the current month, last month and today.
+
+## Installation
+You can manualy install this integration or install using HACS.
+
+### Manual installation
+Copy the custom_component/ngenic folder to your <home assistant folder>/custom_component/ngenic
```

Comment:
add ticks around `custom_components/ngenic` and `<home assistant folder>/custom_components/ngenic`
Also note that it's `custom_components` and not `custom_component` :)